h2o-py/h2o/frame.py

```diff
@@ -1787,22 +1787,24 @@ def group_by(self, by):
         assert_is_type(by, str, int, [str, int])
         return GroupBy(self, by)
 
-    def sort(self, by):
+    def sort(self, by, dir="ASC"):
```

Comment:
This should match Pandas conventions.  So instead of adding a `dir` argument, we should add `ascending` (defaults to `True`).  If you want to leave "ASC" in the Java backend, that's fine, let's just fix it at the client API level.

The other thing is that `Pandas.DataFrame.sort()` is now [deprecated](http://pandas.pydata.org/pandas-docs/stable/whatsnew.html#whatsnew-0170-api-breaking-sorting) and replaced by `.sort_values()` , so we should add that to our Python H2OFrame API as well.  Note that `Pandas.DataFrame.sort()` uses `columns` instead of `by`, but the new API uses `by` just like we do, so that's good!


Example from Pandas:
```
import pandas
df = pandas.DataFrame([{'c1':3,'c2':10},{'c1':2, 'c2':30},{'c1':1,'c2':20},{'c1':2,'c2':15},{'c1':2,'c2':100}])

# sort by a single column
df.sort(columns = 'c2', ascending=False)  #old api
df.sort_values(by = 'c2', ascending=False)  #new api 


#by multiple columns
df.sort(columns = ['c1','c2'], ascending=[False,True])  #old api
df.sort_values(by = ['c1','c2'], ascending=[False,True])  #new api
```